### DISCUSSION TITLE
content-consumption-complexities/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#58] [Announcements]
# content-consumption-complexities/

فرض کنید هر روز وارد فروشگاه مواد غذایی میشید و همون ابتدای فروشگاه رو یه نگاهی می‌اندازید و همون داخل فروشگاه دو تا گاز به چند تا از غذاهای جذاب اول فروشگاه می‌زنید و میاید بیرون!
هیچ وقت هم به انتهای فروشگاه یا کناره‌های فروشگاه (که اتفاقاً مواد غذایی سالم اکثراً اون‌جاها هستند) نمی‌رسید. اینطوری عملاً فقط چیزهایی رو مصرف می‌کنیم که صاحب فروشگاه برامون تدارک دیده و تصمیم گرفته که در ویترین بگذاره.

https://aprd.ir/content-consumption-complexities/

**Comment by YavarMoshirfar** [comment]
اتفاقا یکی از مهم‌ترین مسائل قرن، داشتن رژیم مصرف محتواست. این که چه محتوایی رو چطور و چقدر مصرف کنیم و براش ابزارهای ساده‌ای هم وجود دارند، مثل Innoreader که من سال‌هاست دارم ازش استفاده می‌کنم. 

اما در مورد دور ریختن همچنان سخت و دردآوره.
